lib/osut/utils.rb

```diff
@@ -2054,7 +2054,7 @@ def setpoints(space = nil)
         res[:heating] = nil
         res[:cooling] = nil
       elsif cnd.downcase == "semiheated"
-        res[:heating] = 15.0 if res[:heating].nil?
+        res[:heating] = 14.0 if res[:heating].nil?
```

Comment:
_A first of a handful of changes unrelated to skylights._

The Canadian NECBs use a "< 15°C" threshold - not "<= 15°C" to determine whether a space/zone is considered SEMIHEATED.